lib/models.js

```diff
@@ -48,6 +48,11 @@ exports.sale = {
   shipping: _saleBillingShippingModel,
   transactionDetail: {
     payType: {
+      boleto: {
```

Comment:
I would rather use bankSlip or bankPaymentSlip or bankPayment to this type's name (keep it in english)

Reply:
@rafaelverger this is equivalent to the node boleto in the maxipago documentation, it's not an option to call it something else. You can search for "<boleto>" at the documentation to check it http://developers.maxipago.com/apidocs/maxipago/boleto-bancario/.